spec/rubocop/cop/lint/non_atomic_file_operation_spec.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Lint::NonAtomicFileOperation, :config do
+  shared_examples 'with implicit receiver' do |method_name|
+    it 'does not register an offense' do
+      expect_no_offenses(<<~RUBY)
+        if FileTest.exist?(path)
+          #{method_name}(path)
+        end
+      RUBY
+    end
+  end
```

Comment:
If test L355 exists as a representative case, it may be sufficient for replication, and tests using `shred_context` might be unnecessary.